examples/stable/postgresql-ha/hook-blueprint/README.md

```diff
@@ -0,0 +1,171 @@
+# PostgreSQL-HA
+
+[PostgreSQL](https://www.postgresql.org/) is an object-relational database management system (ORDBMS) with an emphasis on extensibility and on standards-compliance. 
+
+## Introduction
+
+This Helm chart has been developed based on [Postgresql](https://github.com/bitnami/charts/tree/master/bitnami/postgresql) chart but including some changes to guarantee high availability such as:
```

Comment:
if we are not doing this change, we are just installing helm chart with these changes. Do we have to specify this.

Reply:
because I see in the steps to install application we are just installing the chart from upstream.